cuebot/src/main/java/com/imageworks/spcue/DispatchFrame.java

```diff
@@ -72,5 +72,8 @@ public void setMinMemory(long minMemory) {
     public long getMinMemory() {
         return this.minMemory;
     }
+
+    public boolean lokiEnabled;
+    public String lokiURL;
```

Comment:
Please add a comment explaining what what they are and where they are being set

Reply:
done